src/views/Register/components/ValidationRules.vue

```diff
@@ -1,70 +1,104 @@
+                </el-input>
+            </el-form-item>
 
-      <el-form-item prop="password">
-        <el-input
-          v-model="model.password"
-          placeholder="Password"
-          type="password"
-        ></el-input>
-      </el-form-item>
+            <el-form-item prop="password">
+                <el-input
+                        v-model="model.password"
+                        placeholder="Password"
+                        type="password"
+                ></el-input>
+            </el-form-item>
 
-      <el-form-item prop="confirmPassword">
-        <el-input
-          v-model="model.confirmPassword"
-          placeholder="Confirm Password"
-          type="password"
-        ></el-input>
-      </el-form-item>
+            <el-form-item prop="confirmPassword">
+                <el-input
+                        v-model="model.confirmPassword"
+                        placeholder="Confirm Password"
+                        type="password"
+                ></el-input>
+            </el-form-item>
 
-      <el-form-item>
-        <el-button
-          class="register-button"
-          type="primary"
-          native-type="submit"
-          block
-          >Register now</el-button
-        >
-      </el-form-item>
-    </el-form>
-  </el-card>
+            <el-alert type="error" v-model="errors" v-for="error in errors" :key="error[0]">
+                <li>{{error[0]}}</li>
+                <li>{{error[1]}}</li>
+            </el-alert>
+
+            <el-form-item>
+                <el-button
+                        class="register-button"
+                        type="primary"
+                        native-type="submit"
+                        block
+                >Register now
+                </el-button
+                >
+            </el-form-item>
+        </el-form>
+    </el-card>
 </template>
 
 <script>
-import { email, username, password, confirmPassword} from './Rules.js'
-export default {
-  name: "Register",
-  data() {
-    return {
-      model: {
-        username: "",
-        email: "",
-        password: "",
-        confirmPassword: "",
-      },
-      rules: {
-        username,
-        email,
-        password,
-        confirmPassword
-      },
+    import {email, username, password, confirmPassword} from './Rules.js';
+    import axios from "axios";
+
+    export default {
+        name: "Register",
+        data() {
+            return {
+                model: {
+                    username: "",
+                    email: "",
+                    password: "",
+                    confirmPassword: "",
+                },
+                rules: {
+                    username,
+                    email,
+                    password,
+                    confirmPassword
+                },
+                errors: []
+            }
+        },
+        methods: {
+            async register() {
+                let form = this.getForm()
```

Comment:
instead of creating new object this way, you can easily use spread operator `...object` to unpack properties and send directly to request.
`const response = await axios.post("api/auth/register", {...model})`
`{...model}` will create new object with all your properties, which you need for request